tests/formats/png_test.zig

```diff
@@ -1,100 +1,237 @@
-const PixelFormat = @import("../../src/pixel_format.zig").PixelFormat;
-const assert = std.debug.assert;
-const color = @import("../../src/color.zig");
-const ImageReadError = Image.ReadError;
-const png = @import("../../src/formats/png.zig");
+// const assert = std.debug.assert;
+// const testing = std.testing;
+
 const std = @import("std");
-const testing = std.testing;
-const Image = @import("../../src/Image.zig");
 const helpers = @import("../helpers.zig");
+const png = @import("../../src/formats/png.zig");
+const types = @import("../../src/formats/png/types.zig");
+const color = @import("../../src/color.zig");
+const Image = @import("../../src/Image.zig");
+const PixelFormat = @import("../../src/pixel_format.zig").PixelFormat;
+const InfoProcessor = @import("../../src/formats/png/InfoProcessor.zig");
+const ImageReadError = Image.ReadError;
+const expectError = std.testing.expectError;
+const magic_header = types.magic_header;
+
+const valid_header_buf = magic_header ++ "\x00\x00\x00\x0d" ++ png.HeaderData.chunk_type ++
```

Comment:
`valid_header_buf` -> `expected_header`

Reply:
This isn't really expected header, it is just fixture data that a few tests parse into png Header struct that is than checked if it contains expected values.